conf/application.conf

```diff
@@ -191,6 +191,8 @@ external-url {
   reliefs-info = "https://www.gov.uk/hmrc-internal-manuals/capital-gains-manual/cg60201c"
   letting-relief-info = "https://www.gov.uk/tax-sell-home/let-out-part-of-home"
   market-value = "https://www.gov.uk/capital-gains-tax/market-value"
+  tax-free-allowance="https://www.gov.uk/guidance/trusts-and-capital-gains-tax#tax-free-allowance"
```

Comment:
`trusts-tax-free-allowance` here and in the `val` in the config would be more precise